pkg/config/config.go

```diff
@@ -28,6 +31,9 @@ func Get() Config {
 			AllowOrigins:     getEnvSlice("ACCESS_CONTROL_ALLOW_ORIGIN", []string{"*"}, ","),
 			AllowCredentials: getEnvBool("ACCESS_CONTROL_ALLOW_CREDENTIALS", false),
 		},
+		StaticFilesDir: getEnv("STATIC_FILES_DIR", ""),
+		ApiRoutePrefix: "/api", //should start with a "/",
+		ApiVersionPrefix: "/v1",
```

Comment:
Now that I think about it, this is not going to be enough for versioning in my opinion. It is not enough to be hardcoded and also not enough to have as an environment variable. That will just change the string in the endpoint but not actually achieve any versioning at all. There is also a consideration for the OAuth proxy and how it can handle a potentially dynamic URL.

I would remove this entirely for now and we can tackle it later.

Reply:
Removed entirely now :+1: